### ISSUE TITLE
[Bug] Colors do not display in calendar

### ISSUE THREAD

**Issue by ilyesderp** [frappe/erpnext#13135]
Hi, in ERPNext version 10.1.4 colors in calendar are not displayed, for example:
In HR module in Employee Doctype, in the Dashboard there is a calendar based on attendance of employee, when we update attendance of employee it is not reflected on the calendar.
here is a screen shot:
![capture du 2018-02-28 13 47 22](https://user-images.githubusercontent.com/10146310/36788489-6079cd00-1c8e-11e8-86c0-5f7494a6222a.png)
As you see Number of attendances of employee is 4 but nothing is reflected on the calendar.
It used to work perfectly in ERPNext version 9 and older versions.

**Comment by strixaluco** [comment]
Confirming the issue within ERPNext: v11.x.x (5bebbbe) (develop) / Frappe Framework: v11.x.x (f5a6e22) (develop)
But seems the problem is rather that interactions are not being recorded and/or displayed. The lack of colors should be secondary.

**Comment by bghayad** [comment]
Yes I am facing this problem with ERPNext 10.1.4 master and I have been facing it since version 9.
